src/SFA.DAS.ApplyService.Web/Controllers/Roatp/RoatpApplicationController.cs

```diff
@@ -103,46 +103,62 @@ public async Task<IActionResult> Applications()
             var applications = await _apiClient.GetApplications(signinId, false);
             applications = applications.Where(app => app.ApplicationStatus != ApplicationStatus.Rejected).ToList();
 
-            if (!applications.Any())
+            var application = new Apply();
+            Guid applicationId;
+            string applicationStatus;
+
+            if (applications.Any())
             {
-                return await StartApplication(signinId);
+                if (applications.Count > 1)  return View(applications);
+                application = applications.Single();
+                applicationId = application.Id;
+                applicationStatus = application.ApplicationStatus;
             }
+            else
+            {
+                applicationId = await StartApplication(signinId);
+                applicationStatus = ApplicationStatus.InProgress;
```

Comment:
Are we defiantly sure at this point it's InProgress and not New? I'd remove the New check down below as it should never get into that state

Reply:
Newly-created applications are set to InProgress in Qna